.github/workflows/pull-requests.yml

```diff
@@ -51,6 +51,8 @@ jobs:
         run: yarn
       - name: Check formatting
         run: yarn format:check
+      - name: Check for type errors
+        run: yarn typecheck
```

Comment:
1. Move type checking into Linter as described in the other PR.

Reply:
Done in the latest commit.